diff --git a/trunk/user/www/n56u_ribbon_fixed/Advanced_Console_Content.asp b/trunk/user/www/n56u_ribbon_fixed/Advanced_Console_Content.asp
index e20e9de481..4f88e7ca62 100644
--- a/trunk/user/www/n56u_ribbon_fixed/Advanced_Console_Content.asp
+++ b/trunk/user/www/n56u_ribbon_fixed/Advanced_Console_Content.asp
@@ -24,11 +24,7 @@ function initial(){
 	show_menu(5,7,6);
 	show_footer();
 
-	if (!login_safe()){
-		$j('#btn_exec').attr('disabled', 'disabled');
-		$j('#SystemCmd').attr('disabled', 'disabled');
-	}else
-		document.form.SystemCmd.focus();
+	document.form.SystemCmd.focus();
 }
 
 function getResponse(){
@@ -40,8 +36,6 @@ function getResponse(){
 }
 
 function startPost(){
-	if (!login_safe())
-		return false;
 	$j('#btn_exec').attr('disabled', 'disabled');
 	$j.post('/apply.cgi',
 	{
