diff --git a/trunk/user/www/dict/EN.footer b/trunk/user/www/dict/EN.footer
index 1208e46b2f..ebe55c0fc2 100644
--- a/trunk/user/www/dict/EN.footer
+++ b/trunk/user/www/dict/EN.footer
@@ -957,6 +957,10 @@ ConnectionStatus=Connection Status:
 WAN_IP=WAN IP
 Connectiontype=Connection Type
 Gateway=Gateway
+WAN_IP4_Addr=IPv4 Address WAN
+LAN_IP4_Addr=IPv4 Address LAN
+WAN_Gateway=Gateway WAN
+LAN_Gateway=Gateway LAN
 Connected=Connected
 Disconnected=Disconnected
 PauseConnect=Pause
@@ -1189,6 +1193,8 @@ BOP_ctype_item2=ADSL or other connection that requires username and password. It
 BOP_ctype_item3=ADSL or other connection that requires username, password and IP Address. It is known as PPTP.
 BOP_ctype_item4=L2TP requires username, password and IP Address provided by your ISP.
 BOP_ctype_title5=Static IP
+BOP_ctype_pppoe=PPPoE
+BOP_ctype_ipoe=IPoE
 BOP_ctype_item5=Static IP allows your PC to use a fixed IP Address provided by your ISP. This connection type is often used by ADSL service providers.
 WLANConfig11b_SSID_itemdesc=Assign an identification string of up to 32 characters for your wireless connection.
 WLANConfig11b_SSID_itemname=SSID:
